src/main/java/Storage.java

```diff
@@ -0,0 +1,147 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.FileWriter;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.util.List;
+
+import java.nio.file.Paths;
+
+public class Storage {
+    File file;
```

Comment:
Should this variable be private (with a separate getter method to retrieve this field in the class)? 🤔